src/main/java/com/sngular/kloadgen/serializer/CustomStringEnrichedRecordSerializer.java

```diff
@@ -0,0 +1,21 @@
+package com.sngular.kloadgen.serializer;
```

Comment:
I think is is not needed.

Reply:
I don't understand. Isn't the package lin always needed for classes?